rel/files/eunit.config

```diff
@@ -0,0 +1,16 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+[
+    {kernel, [{error_logger, silent}]},
+    {sasl, [{sasl_error_logger, false}]}
```

Comment:
Sorry, I miss your comment here. Idea is good, but actually I would like to see those logs in some file and bounded to specific test to understand what happened for where. It's more complex goal.